experiments/evaluate_config_generator.py

```diff
@@ -78,3 +84,18 @@ def _run_generator(self, cg):
                 run_config, run_config_measurement)
 
             cg.set_last_results([run_config_measurement])
+
+    def _patch_checkpoint_load(self):
```

Comment:
I don't understand this code. Is this a method inside of a method that you are patching?

Reply:
So, the databases don't have a mvnm.  If we run without these changes it hits a key error.

Before, to run these experiments, we had to comment out the code that loaded the mvnm from the database.  This is a proper workaround that patches the code (basically it patches the checkpoint data) to have an empty mvnm.